src/main/java/com/nitnelave/CreeperHeal/block/ExplodedBlockManager.java

```diff
@@ -176,7 +176,7 @@ public static void processExplosion(List<Block> originalBlockList, Location loca
         Bukkit.getPluginManager().callEvent(event);
         if (event.isCancelled())
             return;
-        List<Block> processList = new ArrayList(event.getBlocks());
+        List<Block> processList = new ArrayList<Block>(event.getBlocks());
```

Comment:
Its been awhile, but i think eclipse suggested the edit. 
You probably know this better than me tho...